test/lib/rails-jquery-autocomplete/autocomplete_test.rb

```diff
@@ -62,6 +62,17 @@ class AutocompleteTest < Test::Unit::TestCase
         assert_equal response["label"], "Object Name"
       end
 
+      should 'return an instance of HashWithIndifferentAccess' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.